public/data/Akshay-bendadi.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Akshay Bendadi",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
```

Comment:
Hi @Akshay-bendadi! Since you mentioned being a Computer student in your profile, consider mentioning it here too! :)